gitlab/v4/objects/merge_request_approvals.py

```diff
@@ -119,7 +119,6 @@ def set_approvers(
         approver_ids: Optional[List[int]] = None,
         approver_group_ids: Optional[List[int]] = None,
         approval_rule_name: str = "name",
-        **kwargs: Any,
```

Comment:
The `create()` called in this method supports passing **kwargs so we should pass it on IMO, e.g. for `sudo` and friends, as we document it on the intro page.

https://github.com/python-gitlab/python-gitlab/blob/665b2c8ec6491990bc838139f6afc2797dd895c9/gitlab/v4/objects/merge_request_approvals.py#L217

Reply:
Good catch. Done.